data_api/elasticdump/test_run_elasticdump.py

```diff
@@ -12,25 +13,25 @@
 
 @pytest.mark.parametrize('environ_config, index, expected_url', [
     (
-        {
-            'es_username': 'elastic',
-            'es_password': 'changeme',
-            'es_hostname': 'example.org',
-            'es_port': 9200,
-        },
-        'index_with_no_scheme',
-        'https://elastic:changeme@example.org:9200/index_with_no_scheme'
+            {
+                'es_username': 'elastic',
+                'es_password': 'changeme',
+                'es_hostname': 'example.org',
+                'es_port': 9200,
+            },
+            'index_with_no_scheme',
+            'https://elastic:changeme@example.org:9200/index_with_no_scheme'
```

Comment:
Does this extra indent add anything except cluttering the diff?

Reply:
This is how intellij decided to format the file, will fix.